pylot.py

```diff
@@ -137,6 +137,12 @@ def add_detection_component(graph, bgr_camera_setup, camera_ops, carla_op):
                           [fusion_op])
             graph.connect([fusion_op, carla_op], [fusion_verif_op])
 
+    perfect_tracker_ops = []
```

Comment:
Can you please rename perfect_tracker_ops to tracker_ops, and also store the tracker_op in the list? Thus, we'll return the real object tracker if --obj_tracking flag is enabled, or the perfect tracker if the --perfect_tracking flag is enabled.

Reply:
Added comment indicating why we're keeping them separate for now.